packages/charts/chart-web-components/src/donut-chart/donut-chart.ts

```diff
@@ -217,5 +211,43 @@ export class DonutChart extends FASTElement {
         }
       });
     }
-  };
+
+    this._updateTextInsideDonut();
+  }
+
+  private _getTextInsideDonut(valueInsideDonut: string) {
+    let textInsideDonut = valueInsideDonut;
+
+    if (valueInsideDonut && (this.activeLegend !== '' || this.tooltipProps.isVisible)) {
+      const highlightedDataPoint = this.data.chartData.find(
+        dataPoint =>
+          dataPoint.legend === this.activeLegend ||
+          (this.tooltipProps.isVisible && dataPoint.legend === this.tooltipProps.legend),
+      );
+      textInsideDonut = highlightedDataPoint!.yAxisCalloutData
+        ? highlightedDataPoint!.yAxisCalloutData
+        : highlightedDataPoint!.data.toLocaleString();
+    }
+
+    return textInsideDonut;
+  }
+
+  private _updateTextInsideDonut() {
+    if (!this._textInsideDonut || !this.valueInsideDonut) {
+      return;
+    }
+
+    this._textInsideDonut.textContent = this._getTextInsideDonut(this.valueInsideDonut);
+    const lineHeight = this._textInsideDonut.getBoundingClientRect().height;
+    wrapText(this._textInsideDonut, 2 * this.innerRadius);
+    const lines = this._textInsideDonut.getElementsByTagName('tspan');
+    const start = -Math.trunc((lines.length - 1) / 2);
```

Comment:
is this a minus sign?

Reply:
Yes

Reply:
This is error prone. Make it -1*Math.trunc